virttest/env_process.py

```diff
@@ -385,6 +385,8 @@ def postprocess_vm(test, params, env, name):
         if kill_vm_timeout:
             utils_misc.wait_for(vm.is_dead, kill_vm_timeout, 0, 1)
         vm.destroy(gracefully=params.get("kill_vm_gracefully") == "yes")
+        if params.get("kill_vm_libvirt") == "yes" and params.get("vm_type") == "libvirt":
+            vm.undefine()
```

Comment:
This would undefine vm from main_vm, we should also consider if multi vms are in env

Reply:
got it. yes, it works for me too.